bindata/v3.11.0/openshift-controller-manager/ingress-to-route-controller-clusterrolebinding.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: system:openshift:openshift-controller-manager:ingress-to-route-controller
+roleRef:
+  kind: ClusterRole
+  name: system:openshift:openshift-controller-manager:ingress-to-route-controller
+subjects:
+- kind: ServiceAccount
+  namespace: openshift-infra
+  name: ingress-to-route-controller
```

Comment:
@Miciah why is the ocm-o managing the `ingress-to-route-controller` service account in the `openshift-infra` namespace?

Reply:
My memory is fuzzy, but I think I followed the lead of `separate-sa-role.yaml`.  What namespace should it use?

Reply:
Some poking of the cluster and code led me to figure out why this is necessary:

1. Each controller has its own set of RBAC roles/permissions. To accomplish this, these controllers run as a service account in the `openshift-infra` namespace.
2. Therefore, it's appropriate for OCM-O to manage these cluster roles.

Note @gabemontero we will need to do something similar if we want to narrow the build controller's permissions.